bosh/releases/pre_render_scripts/uaa/uaa/jobs/patch_pre-start.sh

```diff
@@ -36,6 +36,11 @@ patch --verbose "${target}" <<'EOT'
 +        mv /var/lib/ca-certificates/ca-bundle.pem /etc/ssl/certs/"$(basename "${OS_CERTS_FILE}")"
 +      ;;
 +
++      *rhel|centos*)
```

Comment:
```suggestion
+      *rhel|centos|fedora*)
```
Fedora is the same family (and uses the same system).